core/src/test/scala/unit/kafka/javaapi/consumer/ZookeeperConsumerConnectorTest.scala

```diff
@@ -86,19 +83,15 @@ class ZookeeperConsumerConnectorTest extends KafkaServerTestHarness with Logging
                    messagesPerNode: Int,
                    header: String): List[String] = {
     var messages: List[String] = Nil
```

Comment:
looksl like kafka.javaapi.consumer.ZookeeperConsumerConnectorTest.testBasic test is hanging.. Since we are going to remove these tests, maybe we can just disable these tests.

Reply:
I fixed it.